### PR TITLE
Update gradle

### LINKED ISSUE #379: Convert double-quoted strings to single-quoted strings in build.gradle
As mentioned in https://github.com/googleapis/toolkit/pull/378 .

FYI @geigerj @bjwatson

### LINKED ISSUE #1419: Update to latest stable gradle
Gradle 2.7 is pretty old. Fixed in https://github.com/googleapis/toolkit/pull/1417

### PR DESCRIPTION
**Pull request by bootstraponline**
Update protobuf-gradle-plugin to the latest stable version
Fix #379 convert double-quoted strings to single-quoted strings
Fix #1419 Update gradle to the latest stable version

### CODE REVIEW

**Comment by andreamlin** [comment]
After applying this patch to one of my PRs ([passing test log](https://travis-ci.org/googleapis/toolkit/jobs/255408358)), I no longer get the gradle "peer not authenticated" [error](https://travis-ci.org/googleapis/toolkit/jobs/255382496) in the Travis build plaguing the toolkit CI build since yesterday (other failure examples ([1](https://travis-ci.org/googleapis/toolkit/jobs/255391139)) ([2](https://travis-ci.org/googleapis/toolkit/jobs/255332598)) from other authors' PRs).

File: build.gradle [hunk 1]
```diff
@@ -200,8 +200,16 @@ sourceSets {
   }
 }
 
-task setupProtocEnvironment << {
-  test.setEnvironment(PROTOC_COMPILER: project.protobuf.tools.protoc.path)
+task setupProtocEnvironment{
```
Comment (garrettjonesgoogle) [inline build.gradle]: add a space before the curly

File: build.gradle [hunk 2]
```diff
@@ -200,8 +200,16 @@ sourceSets {
...
+    doLast {
+        def protocPath = project.configurations.protobufToolsLocator_protoc.resolve().getAt(0)
+
+        if (protocPath == null) {
+            throw new RuntimeException("protoc path is null")
+        }
+
+        test.setEnvironment(PROTOC_COMPILER: protocPath)
+    }
```
Comment (garrettjonesgoogle) [inline build.gradle]: intent with two spaces instead of four

**Comment by bootstraponline** [comment]
I fixed the spacing.

**Pull request merged**